rai/node/utility.hpp

```diff
@@ -101,7 +101,7 @@ bool fetch_object (T & object, boost::filesystem::path const & path_a, std::fstr
 class mdb_env
 {
 public:
-	mdb_env (bool &, boost::filesystem::path const &);
+	mdb_env (bool &, boost::filesystem::path const &, int max_dbs = 128);
```

Comment:
I'm usually allergic to default args, but in this case it means several hundred edits or other refactoring for what will only be a temporary solution. We should remove this when the wallet store is refactored.